packages/@aws-cdk/aws-events-targets/lib/ecs-task.ts

```diff
@@ -69,6 +69,19 @@ export interface EcsTaskProps {
    * @default A new IAM role is created
    */
   readonly role?: iam.IRole;
+
+  /**
+   * The platform version on which to run your task
+   *
+   * Unless you have specific compatibility requirements, you don't need to specify this.
+   *
+   * More Information:
+   * https://docs.aws.amazon.com/AmazonECS/latest/developerguide/platform_versions.html
+   *
+   * @default A default fargate platform version managed by ECS
```

Comment:
```suggestion
   * @default - ECS will set the Fargate platform version to 'LATEST' 
```
Thank you for not setting it explicitly. Suggestion is assuming this is still describing the default behaviour. The current default message